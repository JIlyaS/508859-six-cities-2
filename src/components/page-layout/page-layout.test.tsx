import * as React from "react";
import * as Enzyme from "enzyme";
import * as Adapter from "enzyme-adapter-react-16";
import toJSON from "enzyme-to-json";

import PageLayout from "./page-layout";

Enzyme.configure({adapter: new Adapter()});

it(`PageLayout correctly renders after relaunch`, () => {
  const children = <div>Контент</div>;
  const tree = Enzyme.shallow(<PageLayout pageName="sign">{children}</PageLayout>);

  expect(toJSON(tree)).toMatchSnapshot();
});
