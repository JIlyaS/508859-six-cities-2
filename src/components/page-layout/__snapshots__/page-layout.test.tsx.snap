// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageLayout correctly renders after relaunch 1`] = `
<div
  className="page page--gray page--login"
>
  <Connect(Header) />
  <div>
    Контент
  </div>
</div>
`;
